packages/components/psammead-most-read/src/list.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string, arrayOf } from 'prop-types';
+import { EasternArabic, WesternArabic } from '@bbc/psammead-locales/numerals';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import Grid from '../../psammead-grid';
+import { MostReadRank, MostReadLink } from './item';
+
+const MostReadItemColumns = {
+  columns: {
+    group0: 5,
+    group1: 5,
+    group2: 5,
+    group3: 5,
+    group4: 5,
+    group5: 5,
+  },
+};
+
+const MostReadRankColumns = {
+  item: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 1,
+    group3: 1,
+    group4: 1,
+    group5: 1,
+  },
+};
+
+const MostReadLinkColumns = {
+  item: true,
+  columns: {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 4,
+  },
+};
+
+const MostReadListColumns = {
+  enableGelGutters: true,
+  enableGelMargins: true,
+  columns: {
+    group0: 1,
+    group1: 1,
+    group2: 2,
+    group3: 2,
+    group4: 2,
+    group5: 5,
+  },
+};
+
+const serviceNumerals = {
+  news: WesternArabic,
+  arabic: EasternArabic,
+};
+
+const StyledOl = styled.ol.attrs({
+  role: 'list',
+})`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`;
+
+const StyledLi = styled.li.attrs({
+  role: 'listitem',
+})`
+  padding: 0.2rem;
+`;
+
+const MostReadItem = ({ service, script, item, rank }) => (
+  <StyledLi>
+    <Grid {...MostReadItemColumns}>
+      <Grid {...MostReadRankColumns}>
+        <MostReadRank service={service} script={script}>
+          {rank}
+        </MostReadRank>
+      </Grid>
+      <Grid {...MostReadLinkColumns}>
+        <MostReadLink service={service} item={item} />
+      </Grid>
+    </Grid>
+  </StyledLi>
+);
+
+const MostReadList = ({ items, service, script }) => {
+  const numerals = serviceNumerals[service];
+  return (
+    <StyledOl>
```

Comment:
There should also be a `dir` prop passed to MostReadList, so that when we have a RTL service, we can pass `dir="rtl"` and the layout will flip around - with the numbers being on the right and the links on the left.
Currently, since we don't pass in `dir`, the layout is the opposite:
<img width="514" alt="Screenshot 2019-11-14 at 13 20 34" src="https://user-images.githubusercontent.com/3028997/68860744-0be2a600-06e2-11ea-85ac-ac2f2e5dc7f0.png">
http://localhost:8180/?path=/story/components-mostread-list--mostreadlist-rtl

Reply:
Resolved - also thanks to you!